docs/book/src/cronjob-tutorial/testdata/project/go.mod

```diff
@@ -4,11 +4,11 @@ go 1.13
 
 require (
 	github.com/go-logr/logr v0.1.0
-	github.com/onsi/ginkgo v1.11.0
-	github.com/onsi/gomega v1.8.1
+	github.com/onsi/ginkgo v1.12.1
+	github.com/onsi/gomega v1.10.1
 	github.com/robfig/cron v1.2.0
-	k8s.io/api v0.18.2
-	k8s.io/apimachinery v0.18.2
-	k8s.io/client-go v0.18.2
-	sigs.k8s.io/controller-runtime v0.6.0
+	k8s.io/api v0.18.6
+	k8s.io/apimachinery v0.18.6
+	k8s.io/client-go v0.18.6
+	sigs.k8s.io/controller-runtime v0.6.2
```

Comment:
Just to share: we have more required changes to do in the docs. See; https://github.com/kubernetes-sigs/kubebuilder/pull/1492


Also, would be nice we are able to automate and check it.